frontend/src/stores/user.js

```diff
@@ -76,10 +76,24 @@ class userStore {
         this.balance = userData.balance;
         this.phoneNumber = userData.phoneNumber;
     }
+    resetCredentials() {
+        this.token = null;
+        this.username = null;
+        this.balance = null;
+        this.phoneNumber = null;
+    }
     setBalance(amount) {
         if (amount > this.balance) throw new Error('Insufficient fund!');
         else this.balance = this.balance - amount;
     }
+    async logout() {
+        try {
+            logoutApi();
```

Comment:
Shouldn't the logoutApi be waited for? Otherwise the error will not be caught.

Reply:
logoutApi does not return a promise so it should not be waited for